Makefile

```diff
@@ -51,6 +52,21 @@ MD_FILES := $(shell \
 		-name '*.md' \
 	| sed 's/ /\\ /g' \
 )
+UI_FILES = $(shell \
```

Comment:
Good point, the name is quite bad – I will try to come up with a better one.
The reason why I am creating an explicit list of files is because we need to exclude tests, docs, reducers, lib/ etc. and work only on strictly ui components.
Probably I could reuse and filter `JS_FILES`? Any other suggestion?